### PR TITLE
Upgrade Swiftlint

### PR DESCRIPTION
**Pull request by captainbarbosa**
Upgrades Swiftlint and ignores the [~`vertical_whitespace`~ `trailing whitespace`](https://github.com/realm/SwiftLint/blob/master/Rules.md#vertical-whitespace) rule - its very verbose when writing code and flags Xcode's default line breaking behavior as not being in conformance.

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ disabled_rules:
   - void_return
   - empty_parentheses_with_trailing_closure
   - comma
+  - trailing_whitespace
```
Comment (friedbunny) [inline .swiftlint.yml]: Hygiene-wise, this is a good rule to keep — if Xcode is hassling you, make sure that you have these settings enabled:

screen_shot_2019-01-02_at_10_51_06_am

**Comment by friedbunny** [comment]
> Upgrades Swiftlint and ignores the [`vertical_whitespace`](https://github.com/realm/SwiftLint/blob/master/Rules.md#vertical-whitespace) rule

I take it this was a typo and disabling [`trailing_whitespace`](https://github.com/realm/SwiftLint/blob/master/Rules.md#trailing-whitespace) is what is being proposed here.

File: .swiftlint.yml (hunk 1 above)
Comment (captainbarbosa) [reply .swiftlint.yml]: Sounds good, I'll drop this from the PR then.

**Comment by captainbarbosa** [comment]
@friedbunny Just upgrading Swiftlint for now then 😄

File: Podfile.lock [hunk 2]
```diff
@@ -52,6 +52,6 @@ SPEC CHECKSUMS:
   SwiftLint: 6772320e40b52049053a518c17db9b0634a0b45a
   Turf: c6bdf62d6a70c647874f295dd1cf4eefc0c3e9e6
 
-PODFILE CHECKSUM: 3b8d3b246ccf3e793538f5d67a493c634bde9758
+PODFILE CHECKSUM: 27a37682984bb1f29da3f01c40810fe7bcfa6310
 
-COCOAPODS: 1.6.0.beta.2
+COCOAPODS: 1.6.0.beta.1
```
Comment (friedbunny) [inline Podfile.lock]: @captainbarbosa Hrm, you might upgrade your CocoaPods install.

**Pull request merged**